modules/tinymce/src/core/main/ts/api/Editor.ts

```diff
@@ -449,6 +449,10 @@ class Editor implements EditorObservable {
     return getParam(this, name, defaultVal, type);
   }
 
+  public hasPlugin(key: string): boolean {
```

Comment:
This will need documentation since it's a public API. I'd suggest talking to @tylerkelly13 to get a hand.